src/frequenz/sdk/timeseries/_moving_window.py

```diff
@@ -138,6 +138,13 @@ def __init__(  # pylint: disable=too-many-arguments
         self._resampler_sender: Sender[Sample] | None = None
         self._resampler_task: asyncio.Task[None] | None = None
 
+        self._wait_for_num_samples: int = 0
+        """The number of samples to wait for before the wait_for_num_samples channels
+        sends out an event."""
+        self._wait_for_samples_channel = Broadcast[None](
+            "Wait for number of samples channel."
+        )
```

Comment:
So the text here is for debugging purposes only, right @shsms?

I'd suggest using a shorter string for that and using the long string as documentation for the variable:

```suggestion
        self._wait_for_samples_channel = Broadcast[None]("wait-for-samples")
        """Channel to send events to when wait for number of samples is triggered."""

```